admin/controllers/school.js

```diff
@@ -467,8 +487,13 @@ const savePupilNotTakingCheck = async (req, res, next) => {
   }
 
   const todayDate = moment(moment.now()).format()
-  const pupilsData = await fetchMultiplePupils(Object.values(req.body.pupil))
+  const postedPupils = req.body.pupil
+  const pupilsData = await fetchMultiplePupils(Object.values(postedPupils))
+
   let pupilsToSave = []
+  let pupilsList
+  let attendanceCodes
+  let flashMessage = 'Reasons updated'
```

Comment:
'Reason updated' is not needed as value right?
that can just be let flashMessage